WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -660,6 +725,62 @@ class PostListViewModel @Inject constructor(
         }
     }
 
+    private fun loadPostFromRemote(localPostId: Int) {
+        // We need network connection to load a remote post
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        val post = postStore.getPostByLocalPostId(localPostId)
+        if (post != null) {
+            originalPostCopyForConflictUndo = post.clone()
+            dispatcher.dispatch(PostActionBuilder.newFetchPostAction(RemotePostPayload(post, site)))
+            _toastMessage.postValue(ToastMessageHolder(R.string.toast_conflict_updating_post, Duration.SHORT))
+        } else {
+            localPostIdForConflictResolutionDialog = null
+        }
+    }
+
+    private fun savePostFromLocal(localPostId: Int) {
```

Comment:
Similar to my comment for the `loadPostFromRemote` method, I don't think the name exactly matches what this method does. How about `updateConflictedPostWithItsLocalVersion`? I am not too sure `update` is the best term to use in either of these names, but I couldn't find anything better.

Reply:
I think your suggestion still describes the purpose more accurately so, agreed and committed the change in 77b8abddbb